### PR TITLE
Some urgently needed edits and modifications

### PR DESCRIPTION
**Pull request by smileBeda**
- Removed unnecessary links to dead or duplicate pages
- Added new sections for develop WITH and renamed main develop(with)ClassicPress 
- Moved browser support to faq
- Prepared for taking link to actual codex.

### CODE REVIEW

File: developing-with-classicpress/index.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Developing With ClassicPress
+
+- <span style="font-size: 150%">[Codex](https://codex.classicpress.net/)</span><!--this should link to the new, not-yet-existing DOC for Code page-->
```
Comment (striebwj) [inline developing-with-classicpress/index.md:3]: Could we remove this for now until this is up

File: developing-with-classicpress/index.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+- <span style="font-size: 150%">[Security Page](https://docs.classicpress.net/developing-with-classicpress/security-page/)</span><!--this will only work after renaming developing-classicpress to developing-with-classicpress-->
```
Comment (striebwj) [inline developing-with-classicpress/index.md:4]:
```suggestion
- <span style="font-size: 150%">[Security Page](https://docs.classicpress.net/developing-with-classicpress/security-page/)</span>
```

File: developing-with-classicpress/index.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+- <span style="font-size: 150%">[Dev Recipes](https://docs.classicpress.net/dev-recipes/)</span><!--this will only work after actually creating the DEV Recipes-->
```
Comment (striebwj) [inline developing-with-classicpress/index.md:5]:
```suggestion
- <span style="font-size: 150%">[Dev Recipes](https://docs.classicpress.net/dev-recipes/)</span>
```

File: developing-with-classicpress/index.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+- <span style="font-size: 150%">[Tools and Standards](https://docs.classicpress.net/tools-and-standards)<!--this will only work after actually creating the Tools and Standards Page-->
```
Comment (striebwj) [inline developing-with-classicpress/index.md:6]:
```suggestion
- <span style="font-size: 150%">[Tools and Standards](https://docs.classicpress.net/tools-and-standards)
```

File: developing-with-classicpress/tools-and-standards [hunk 5]
```diff
@@ -0,0 +1,14 @@
+# Tools and Standards
+
+## Recommended Tools for Development
+
+- [PHPStorm](https://www.jetbrains.com/phpstorm/)
+- [SublimeText](https://www.sublimetext.com)
```
Comment (striebwj) [inline developing-with-classicpress/tools-and-standards:6]: I am not sure about sharing code editors, there are a lot and I don't really want to get into the game of having to update this with every single one. Thoughts?

**Comment by striebwj** [comment]
If we are merging this PR over the others, could the close the others. 

One other note, will changing the urls will lead to those links becoming 404s. If so, we should handle those.

File: developing-with-classicpress/index.md (hunk 2 above)
Comment (smileBeda) [reply developing-with-classicpress/index.md:4]: Agreed. Left the comment only for info purpose.

File: developing-with-classicpress/index.md (hunk 3 above)
Comment (smileBeda) [reply developing-with-classicpress/index.md:5]: Agreed.

File: developing-with-classicpress/index.md (hunk 4 above)
Comment (smileBeda) [reply developing-with-classicpress/index.md:6]: Yes :)

File: developing-with-classicpress/tools-and-standards (hunk 5 above)
Comment (smileBeda) [reply developing-with-classicpress/tools-and-standards:6]: Valid point.

**Comment by smileBeda** [comment]
This is obsolete - closing.

**Pull request closed without merging**